Gem/Code/Source/Components/UI/UiSettingsComponent.h

```diff
@@ -40,8 +40,7 @@ namespace MultiplayerSample
         void Deactivate() override;
     private:
         // WindowNotificationBus overrides
-        void OnWindowResized(uint32_t width, uint32_t height) override;
```

Comment:
Is OnWindowResized not needed?

Reply:
Correct. It was previously being used as a poor/indirect way to detect fullscreen changes, though it only actually worked if going to fullscreen changed the size of the window. 

I've added an OnFullScreenModeChanged() notification that we're now using instead, which provides a reliable way to detect the state change.